docs/ref/contrib/admin/index.txt

```diff
@@ -986,6 +986,19 @@ subclass::
         The ``FieldListFilter`` API is considered internal and might be
         changed.
 
+      You can provide a title to use instead of the default, useful for fields
+      on a related model where the default title would be confusing or not make
+      sense without the context of the related model, for example::
+
+          class BookAdmin(admin.ModelAdmin):
+              list_filter = (
+                  ('author__name', 'Author Name'),
+              )
```

Comment:
It strikes me that using a string might actually be more useful that giving the `FieldListFilter` subclass directly. As such, I'd be inclined to put this example first. 

Something like: 

1. End `a tuple...` paragraph with `...for example, with a string title::`
2. This `BookAdmin` code example. 
3. Sentence saying, `...useful for fields on a related model...`
4. "...Or with a `FieldListFilter` subclass":
5. Existing examples and discussion. 
6. The `versionadded` note below.